bugtool/cmd/root.go

```diff
@@ -413,6 +428,42 @@ func split(prompt string) (string, []string) {
 	return cmd, args
 }
 
+// hashIPseckeys hashes keys and replaces them with its sha256
+func hashIPseckeys(output string) string {
+	var final []string
+	// splitting the result as an array of string
```

Comment:
Prefer the infinitive in comments, and please respect the punctuation: lower case at the beginning, periods at the end (for long sentences in particular).

```suggestion
	// Split the result as a slice of strings.
```

Also note that you do not need to comment each step. The code should be “self-explanatory”, easy enough to understand without too many comments. Document the tricky parts :).